### PR TITLE
Feature/byron address support

### PR DESCRIPTION
**Pull request by MMortaraAtix**
# Description

_A clear and concise description of what this pull request does or fixes._

# Proposed Solution

_**Optional** Explain how does this PR solves the problem stated in [Description](#Description). You can also enumerate different alternatives considered while approaching this task._

# Important Changes Introduced

_**Optional** Notice Reviewers about changes that were introduced while developing this task_

# Testing

_**Optional** Leave some recommendations should be useful while reviewers are testing this PR_

### CODE REVIEW

File: cardano-rosetta-server/src/server/controllers/construction-controller.ts [hunk 1]
```diff
@@ -195,6 +195,7 @@ const configure = (
           parseInt(ttl)
         );
         const payloads = constructPayloadsForTransactionBody(unsignedTransaction.hash, unsignedTransaction.addresses);
+        // test only
```
Comment (jvieiro) [inline cardano-rosetta-server/src/server/controllers/construction-controller.ts:198]:
```suggestion
```

File: cardano-rosetta-server/test/e2e/construction/construction-parse-api.test.ts [hunk 2]
```diff
@@ -69,6 +71,26 @@ describe(CONSTRUCTION_PARSE_ENDPOINT, () => {
     ]);
   });
 
+  test('Should return 1 input, 2 outputs and signers if a valid signed transaction is set', async () => {
```
Comment (jvieiro) [inline cardano-rosetta-server/test/e2e/construction/construction-parse-api.test.ts:74]:
```suggestion
  test('Should return input with byron address and signers if a valid signed transaction is set', async () => {
```

**Review by jvieiro — CHANGES_REQUESTED** [review]

File: cardano-rosetta-server/test/e2e/construction/construction-parse-api.test.ts (hunk 2 above)
Comment (MMortaraAtix) [reply cardano-rosetta-server/test/e2e/construction/construction-parse-api.test.ts]: > 

Done, pls check it @jvieiro

File: cardano-rosetta-server/src/server/controllers/construction-controller.ts (hunk 1 above)
Comment (MMortaraAtix) [reply cardano-rosetta-server/src/server/controllers/construction-controller.ts]: > 

Done, pls check it @jvieiro

File: cardano-rosetta-server/README.md [hunk 3]
```diff
@@ -154,3 +154,7 @@ test('Launchpad', async () => {
 
 - Introduce a way to check current schema in order to throw an error if `cardano-db-sync` is updated but it doesn't match the snapshot.
 - Automate snapshots generation as it currently is a semi-manual process.
+
+
+### Notes
+- New Byron Address support.
```
Comment (tomasBustamante) [inline cardano-rosetta-server/README.md:160]: What's the purpose of this here? :thinking: 
Bear in mind that there's a [documentation section in the project](https://github.com/input-output-hk/cardano-rosetta/tree/master/docs) where you could add more details if needed.

**Pull request merged**